src/flavors/__tests__/strict.test.ts

```diff
@@ -0,0 +1,78 @@
+import { plainToClass } from 'class-transformer';
+import { validate } from 'class-validator';
+
+import * as strict from '../strict';
+import { INPUT, createFixtures } from './fixtures';
+
+describe('flavors.strict', () => {
+    const Example = createFixtures(strict);
+
+    it('validates required fields', async () => {
+        const obj = plainToClass(Example, {});
+
+        // expect every field to exist, but be undefined
+        expect(Object.keys(obj)).toHaveLength(33);
+        for (const value of Object.values(obj)) {
+            expect(value).toBeUndefined();
```

Comment:
As we've discovered, using `@Expose` means that decorated fields will have an `undefined` value (rather than be omitted) if no data is provided.